<?xml version="1.0"?>
<launch>

  <node pkg="perception" type="perception_node" respawn="false" name="perception" output="screen">
    <rosparam file="$(find perception)/config/perception_config.yaml" command="load" />
    <param name="config_path" value="$(find perception)/config/bisenetv1_SUPS.py" />
    <param name="weight_path" value="$(find perception)/models/model_final.pth" />
<!--     <remap from="input_image" to="/camera/image_raw" /> -->
  </node>

  <node name="rviz_perception" pkg="rviz" type="rviz" args="-d $(find perception)/src/ros_viewer/perception.rviz"/>
  
</launch>